Giftson/randomNumGuesser.py

```diff
@@ -0,0 +1,23 @@
+"""
+Python Program to guess the number within the range 1-10
+Uses the library random
+Program Exp
+"""
+import random
+
+# using randint function from the random library to generate integer values
+# randint(1,10) produces a random value within the range of 1 to 10
+n = random.randint(1, 10)
+
+print("Please Guess the Number!")
+
+# loop will continue to run until the break statement runs.
+while True:
+    o = int(input())
+    if n < o:
+        print("The number you entered is larger\n Please guess the number again")

```

Comment:
add like hint: this comment same for other hint also.

Reply:
anna im not getting what you are trying to say here anna

Reply:
print("hint: <your statement>")

Reply:
> print("hint: ")

ow okay i get it now